torch/testing/_internal/common_nn.py

```diff
@@ -2953,7 +2944,7 @@ def _multimarginloss_reference(input, target_idx, p, margin, weight):
     output = 0
     for i in range(0, len(input)):
         if i != target_idx:
-            output += max(0, weight[target_idx] * (margin - input[target_idx] + input[i]) ** p)
+            output += weight[target_idx] * (max(0, (margin - input[target_idx] + input[i])) ** p)
```

Comment:
This reference function was implemented wrongly, see docs
<img width="701" alt="Screenshot 2023-12-11 at 3 38 01 PM" src="https://github.com/pytorch/pytorch/assets/35276741/2dbbe71f-db4b-4324-92c2-5f299d9ea579">
and the implementation

https://github.com/pytorch/pytorch/blob/585aea6e770f601159e85fa4cc39ce7f67155fdf/aten/src/ATen/native/LossMultiMargin.cpp#L37-L45